### PR TITLE
Fix enums defined in classes

### PR DESCRIPTION
**Pull request by varunagrawal**
This PR fixes adding of enums defined under classes.

### CODE REVIEW

File: gtwrap/pybind_wrapper.py
Comment (gchenfc) [inline gtwrap/pybind_wrapper.py:310]: why not just name all the classes?  Is there any downside to giving the class a variable name?

File: gtwrap/pybind_wrapper.py
Comment (gchenfc) [inline gtwrap/pybind_wrapper.py:288]: just double checking, is there any chance of this throwing a "Nonetype is not iterable" error?
i.e. hopefully the default value of enums is `[]` rather than `None`?

**Review by gchenfc — APPROVED** [review]
LGTM if it passes the unit tests (didn't run on my local machine).

I don't see the reason why we don't just give all the classes a variable name, but I will not block merging.

File: gtwrap/pybind_wrapper.py
Comment (varunagrawal) [reply gtwrap/pybind_wrapper.py:310]: There isn't other than a lack of time. I want to push this out fast since there are other things that need my attention...

File: gtwrap/pybind_wrapper.py
Comment (varunagrawal) [reply gtwrap/pybind_wrapper.py:288]: In `Class` in `classes.py` we have `self.enums = []` so all good :slightly_smiling_face:

**Pull request merged**
